docs/sops/serverless-controller-knative-up.md

```diff
@@ -0,0 +1,93 @@
+# OpenShift Serverless Knative Serving Control Plane Readiness
+
+## Severity: Critical
+
+## Impact
+
+Any users or services using Knative Serving will be unable to get their Knative Services reconciled.
+This means, any new Knative Services might not be created, and any existing Knative Services might not be updated. Also, it might also mean that the user workload is not being served.
+
+## Summary
+
+OpenShift Serverless does not create any alerts out-of-the box. However, users can create alerts based on the metrics exposed.
+
+The metric `knative_up{namespace="openshift-serverless", type="serving_status"}` should always have a value of `1`. 
+
+This failure should only occur in extremely catastrophic scenarios as all components are deployed in HA configuration.
+
+Possible causes can be:
+- Network issues in the cluster
+- Resource starvation on the cluster
+- OpenShift Serverless upgrade issues
+
+You should check both OpenShift Serverless operator and Knative Serving control plane status.
+
+## Steps
+
+1. Check to see if all the pods are running in `openshift-serverless` namespace:
+
+```bash
+$ oc -n openshift-serverless get pods
+```
+
+2. If they are not running, look at the pod's logs/events to see what may be causing the issues. Please make sure to grab the logs/events so they can be shared with the engineering team later:
+
+```bash
+# Get Pod names as we don't have a common label
```

Comment:
should we introduce such a label? (wondering about this being scriptable)

Reply:
Created https://issues.redhat.com/browse/SRVCOM-2703